cfgov/unprocessed/js/modules/util/data-set.js

```diff
@@ -19,7 +21,9 @@ function _toCamelCase( str ) {
  * @returns {Object} The data set.
  */
 function dataSet( element ) {
-  if ( document.documentElement.dataset ) return element.dataset;
+  if ( document.documentElement.dataset ) {
+    return _assign({}, element.dataset );
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1783/commits/03e64c145f02123a25f7dde4f447bbc5411df6b3.